src/components/2020/uikit/Tag.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import React from "react";
+import cx from "classnames";
+
+type Props = {
+  children: React.Node,
+  color: "black" | "white" | "purple" | "cyan" | "aqua",
```

Comment:
i was actually passing a prop called className so we can style the tag from whoever uses it, but this is also good :)

Reply:
The idea behind this over-abstraction is to force us to not deviate from the design document. Passing a `className` opens the door to one-offs, and I'd like to avoid that as much as possible.